plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerTypeMapping.java

```diff
@@ -246,4 +350,19 @@ private DataSetup sqlServerCreateAndInsert(String tableNamePrefix)
         properties.setProperty("password", sqlServer.getPassword());
         return new CreateAndInsertDataSetup(new JdbcSqlExecutor(sqlServer.getJdbcUrl(), properties), tableNamePrefix);
     }
+
+    private static void checkIsGap(ZoneId zone, LocalDateTime dateTime)
```

Comment:
I can't find the caller of the method in this PR.

Reply:
Me neither, removed!

Reply:
Don't remove. Search the other classes. I'm sure one of those has a @BeforeClass setup method that verifies something.

It's useful imo.

Reply:
Those are used in tests for timestamp or time with time zone, to verify that tested TZs actually have gaps. This connector doesn't support such types for now.

Reply:
Thanks for checking. 👍